A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Misc.xml

```diff
@@ -294,4 +294,56 @@
             <TOOLTIPID>%((A:APU PCT RPM, Percent) 0 &gt; (A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &lt; and)%{if}TT:COCKPIT.TOOLTIPS.UNAVAILABLE%{end} %((A:APU PCT RPM, Percent) #AVAIL_PERCENTAGE# &gt;= (A:APU SWITCH, Bool) and)%{if}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_OFF%{else}TT:COCKPIT.TOOLTIPS.APU_STARTER_TURN_ON%{end}</TOOLTIPID>
         </UseTemplate>
     </Template>
+        <Template Name="FBW_ENGINE_Switch_Master_SubTemplate">
+		<DefaultTemplateParameters>
+			<NODE_ID>ENGINE_Switch_Master_#ID#</NODE_ID>
+			<ANIM_NAME>ENGINE_Switch_Master_#ID#</ANIM_NAME>
+			<WWISE_EVENT_1>engine_master_switch_on</WWISE_EVENT_1> 
+			<WWISE_EVENT_2>engine_master_switch_off</WWISE_EVENT_2> 
+			<PART_ID>ENGINE_Switch_Master</PART_ID>
+			<Switch Param="TYPE">
+				<Case Value="LOCKED">
```

Comment:
You should be able to remove this switch statement, since the airbus master switch isn't locked.

Reply:
WIll remove